components/Sponza.vue

```diff
@@ -3,8 +3,20 @@
     <p class="sponsor-title">
       Sponsors
     </p>
+    <div id="sponzaButtonContainer">
+      <img
+        src="~/assets/graphics/leafLeft.svg"
+      >
+      <a href="mailto:logistics@nwplus.io?subject=CMD-F Sponsorship">
+        <button id="sponzaButton">
```

Comment:
I was going to say make this a component, but the two buttons have different hover states... 🤯

Reply:
oh i think i forgot the hover states too lol

Reply:
@ianmah I fixed the hover state, put it into its own component, and switched it to an <a> tag so its more accessible, can you gimme approve